lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -31,15 +31,16 @@
   record_type:
     description:
     - The type of DNS record name
-    - Currently, 'A', 'AAAA', and 'PTR' are supported
+    - Currently, 'A', 'AAAA', 'PTR' and 'TXT' are supported
```

Comment:
As @Akasurde mentioned, please udpate this line e.g. to let users now, the TXT feature is since ansible version 2.5:
~~~diff
-- Currently, 'A', 'AAAA', 'PTR' and 'TXT' are supported
+- Currently, 'A', 'AAAA', 'PTR' and 'TXT' (added in version 2.5) are supported
~~~